files/justfiles/secureblue.just

```diff
@@ -730,6 +736,8 @@ dns-selector:
     readonly policy_file="/etc/chromium/policies/managed/10-securedns-browser.json"
     mkdir -p /etc/systemd/resolved.conf.d/
     mkdir -p /etc/chromium/policies/managed/
+    restorecon -R /etc/systemd/resolved.conf.d
```

Comment:
Is this necessary? After using the dns-selector, running `restorecon -Rv /etc/systemd/resolved.conf.d` does not report any changes